### DISCUSSION TITLE
SiS 6326 Emulation?

### DISCUSSION THREAD

**Discussion by senseab** [PCBox/PCBox#103] [Ideas]
Here's 2 documents. Are these datasheet?

[Information about SiS 6326](https://www.vgamuseum.info/index.php/cpu/item/393-sis-6326-agp)

[6326.documentation.pdf](https://github.com/user-attachments/files/16901820/6326.documentation.pdf)
[393_sis6326.pdf](https://github.com/user-attachments/files/16901821/393_sis6326.pdf)

**Comment by bachimusprime** [comment]
That seems good enough to add it, but I'd like to ask what new things adding this would let you do in PCBox, because this is only a Direct3D 5 card. It'd need to do Direct3D 7 to be worthwhile to add. I've got other stuff to work on anyway.

**Reply by senseab** [reply]
My first computer used SiS 6326. It worked for Direct3D 7. And also for DX8 or DX9

**Reply by bachimusprime** [reply]
I'm pretty sure it didn't have all of the DX7+ features though, and that's my point.

**Reply by Torinde** [reply]
Probably everything above DX5 was in software mode, without acceleration?

**Reply by Torinde** [reply]
Could that be useful as fist step to eventually reach the SiS chips supporting DX7/8/9?

Separately, what are your thoughts about adding support for [SoftGPU](https://github.com/JHRobotics/softgpu/issues/13) via:
- VMWare SVGA-II (VMWare Workstation/Player, VirtulBox: VMSVGA, QEMU: vmware-svga) 
- VBox SVGA (VirtulBox: VBoxSVGA)
- [QEMU virtio](https://www.kraxel.org/blog/2019/09/display-devices-in-qemu/) (for [this driver](https://github.com/virtio-win/kvm-guest-drivers-windows/pull/943) and [potentially SoftGPU](https://github.com/JHRobotics/softgpu/issues/13#issuecomment-1628424754))?

**Reply by senseab** [reply]
> Probably everything above DX5 was in software mode, without acceleration?

It might be possible.
